src/app/components/Hero/index.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
```

Comment:
This is not needed here as the component will be used in a component that already declares it